Sources/ScreenObject/ScreenObject.swift

```diff
@@ -83,6 +93,21 @@ open class ScreenObject {
         return self
     }
 
+    @discardableResult
+    func waitForFirstScreenElement() throws -> Self {
```

Comment:
Nitpick - if we decide to stick with two functions, maybe can rename this to `waitForFirstElementOnScreen` or `waitForFirstElement`, I kept reading it as first screen.